components/ordhook-cli/src/config/file.rs

```diff
@@ -1,29 +1,24 @@
 use ordhook::chainhook_sdk::indexer::IndexerConfig;
+use ordhook::chainhook_sdk::observer::DEFAULT_INGESTION_PORT;
 use ordhook::chainhook_sdk::types::{
     BitcoinBlockSignaling, BitcoinNetwork, StacksNetwork, StacksNodeConfig,
 };
 use ordhook::config::{
-    BootstrapConfig, Config, LimitsConfig, LogConfig, PredicatesApi, PredicatesApiConfig,
-    StorageConfig,
+    Config, LogConfig, PredicatesApi, PredicatesApiConfig, ResourcesConfig, SnapshotConfig,
+    StorageConfig, DEFAULT_BITCOIND_RPC_THREADS, DEFAULT_BITCOIND_RPC_TIMEOUT,
+    DEFAULT_CONTROL_PORT, DEFAULT_MEMORY_AVAILABLE, DEFAULT_ULIMIT,
 };
 use std::fs::File;
 use std::io::{BufReader, Read};
 
-pub const DEFAULT_INGESTION_PORT: u16 = 20455;
-pub const DEFAULT_CONTROL_PORT: u16 = 20456;
-pub const STACKS_SCAN_THREAD_POOL_SIZE: usize = 10;
-pub const BITCOIN_SCAN_THREAD_POOL_SIZE: usize = 10;
-pub const STACKS_MAX_PREDICATE_REGISTRATION: usize = 50;
-pub const BITCOIN_MAX_PREDICATE_REGISTRATION: usize = 50;
-
 #[derive(Deserialize, Debug, Clone)]
 pub struct ConfigFile {
     pub storage: StorageConfigFile,
     pub http_api: Option<PredicatesApiConfigFile>,
-    pub limits: LimitsConfigFile,
+    pub resources: ResourcesConfigFile,
     pub network: NetworkConfigFile,
     pub logs: Option<LogConfigFile>,
-    pub bootstrap: Option<BootstrapConfigFile>,
+    pub snapthot: Option<SnapshotConfigFile>,
```

Comment:
I wonder if this should be `snapshot` instead of `snapthot`?

Reply:
Thanks for catching this @GuutBoy! Addressed with https://github.com/hirosystems/ordhook/pull/250/commits/eca79f0e92ae82843f55d2e270a02874ad4e1120